src/runtime/virtcontainers/utils/utils_test.go

```diff
@@ -297,9 +297,9 @@ func TestBuildSocketPath(t *testing.T) {
 		msg := fmt.Sprintf("test[%d]: %+v", i, d)
 
 		if d.valid {
-			assert.NoErrorf(err, "test %d, data %+v", i, d, msg)
```

Comment:
Nit: how about deleting `"test %d, data %+v", i, d,` and keep `msg`?

Reply:
ok, thanks.

Reply:
The previous PR is closed by  misoperation, please review again. @liubin

Reply:
resubmit the pull request at  https://github.com/kata-containers/kata-containers/pull/3718